src/test/java/gregtech/common/covers/CoverFluidRegulatorTest.java

```diff
@@ -0,0 +1,227 @@
+package gregtech.common.covers;
+
+import gregtech.api.capability.impl.*;
+import net.minecraft.init.*;
+import net.minecraft.util.*;
+import net.minecraftforge.fluids.*;
+import net.minecraftforge.fluids.capability.*;
+import org.junit.*;
+
+import java.util.function.*;
+
+import static org.junit.Assert.*;
+
+public class CoverFluidRegulatorTest {
```

Comment:
The tabs/spaces thing is probably just because it was a new file, and I got used to relying on IntelliJ's detection of existing indent and didn't notice it was different. I can fix that shortly.